src/overlays/actors/ovl_En_Kendo_Js/z_en_kendo_js.c

```diff
@@ -71,41 +71,125 @@ static ColliderCylinderInit sCylinderInit = {
-    { &object_js_Anim_0003DC, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -8.0f }, // ENKENDOJS_ANIM_3
-    { &object_js_Anim_000AD4, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -8.0f }, // ENKENDOJS_ANIM_4
+    { &object_js_Anim_SlappingKnee, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -8.0f }, // unused
+    { &object_js_Anim_Default, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -8.0f },
+    { &object_js_Anim_Flailing, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -8.0f },
+    { &object_js_Anim_StopFlailing, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -8.0f },
+    { &object_js_Anim_Scared, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -8.0f },
 };
 
-s16 D_80B27CE0[][3] = {
-    { 0x2731, 0x2732, 0x2733 },
-    { 0x2734, 0x2735, 0x2736 },
-    { 0x2737, 0x2738, 0x2739 },
+typedef enum EnKendoJsMessage {
+    ENKENDOJS_MSG_HUMAN_INTRO_DAY1 = 0x2710,
+    ENKENDOJS_MSG_HUMAN_INTRO_SHORT_DAY1 = 0x2711,
+    ENKENDOJS_MSG_HUMAN_INTRO_DAY2 = 0x2712,
+    ENKENDOJS_MSG_HUMAN_INTRO_SHORT_DAY2 = 0x2713,
+    ENKENDOJS_MSG_HUMAN_INTRO_DAY3 = 0x2714,
+    ENKENDOJS_MSG_HUMAN_INTRO_SHORT_DAY3 = 0x2715,
+    ENKENDOJS_MSG_WHICH_COURSE = 0x2716,
+    ENKENDOJS_MSG_CANCEL = 0x2717,
+    ENKENDOJS_MSG_NOT_ENOUGH_RUPEES = 0x2718,
+    ENKENDOJS_MSG_NOVICE_COURSE_SELECTED = 0x2719,
+    ENKENDOJS_MSG_NOVICE_COURSE_START = 0x271A,
+    ENKENDOJS_MSG_NOVICE_COURSE_RETRY_ROUND1 = 0x271B,
+    ENKENDOJS_MSG_NOVICE_COURSE_PASS_ROUND1 = 0x271C,
+    ENKENDOJS_MSG_NOVICE_COURSE_RETRY_ROUND2 = 0x271D,
+    ENKENDOJS_MSG_NOVICE_COURSE_PASS_ROUND2 = 0x271E,
+    ENKENDOJS_MSG_NOVICE_COURSE_RETRY_ROUND3 = 0x271F,
+    ENKENDOJS_MSG_NOVICE_COURSE_PASS_ROUND3 = 0x2720,
+    ENKENDOJS_MSG_NOVICE_COURSE_RETRY_ROUND4 = 0x2721,
+    ENKENDOJS_MSG_NOVICE_COURSE_PASS_ROUND4 = 0x2722,
+    ENKENDOJS_MSG_NOVICE_COURSE_RETRY_ROUND5 = 0x2723,
+    ENKENDOJS_MSG_NOVICE_COURSE_PASS_ROUND5 = 0x2724,
+    ENKENDOJS_MSG_NOVICE_COURSE_RETRY_ROUND6 = 0x2725,
+    ENKENDOJS_MSG_NOVICE_COURSE_PASS_ROUND6 = 0x2726,
+    ENKENDOJS_MSG_NOVICE_COURSE_RETRY_ROUND7 = 0x2727,
+    ENKENDOJS_MSG_NOVICE_COURSE_PASS_ROUND7 = 0x2728,
+    ENKENDOJS_MSG_NOVICE_COURSE_WRONG = 0x2729,
+    ENKENDOJS_MSG_NOVICE_COURSE_SHEATH_WEAPON = 0x272A,
+    ENKENDOJS_MSG_NOVICE_COURSE_NO_Z_TARGET = 0x272B,
+    ENKENDOJS_MSG_NOVICE_COURSE_NO_SWORD = 0x272C,
+    ENKENDOJS_MSG_EXPERT_COURSE_PERFECT = 0x272D,
+    ENKENDOJS_MSG_EXPERT_COURSE_INSUFFICIENT = 0x272E,
+    ENKENDOJS_MSG_EXPERT_COURSE_ACCEPT_HEART_PIECE = 0x272F,
+    ENKENDOJS_MSG_EXPERT_COURSE_ACCEPT_RED_RUPEE = 0x2730,
+    ENKENDOJS_MSG_DEKU_DAY1 = 0x2731,
+    ENKENDOJS_MSG_DEKU_DAY2 = 0x2732,
+    ENKENDOJS_MSG_DEKU_DAY3 = 0x2733,
+    ENKENDOJS_MSG_GORON_DAY1 = 0x2734,
+    ENKENDOJS_MSG_GORON_DAY2 = 0x2735,
+    ENKENDOJS_MSG_GORON_DAY3 = 0x2736,
+    ENKENDOJS_MSG_ZORA_DAY1 = 0x2737,
+    ENKENDOJS_MSG_ZORA_DAY2 = 0x2738,
+    ENKENDOJS_MSG_ZORA_DAY3 = 0x2739,
+    ENKENDOJS_MSG_EXPERT_COURSE_SELECTED = 0x273A,
+    ENKENDOJS_MSG_EXPERT_COURSE_RULES = 0x273B,
+    ENKENDOJS_MSG_DONT_WANT_TO_DIE = 0x273C,
+} EnKendoJsMessage;
```

Comment:
One big problem of introducing text/message enums in individual actors is having to redefine them in other files that references those same ids, names can get out of sync and it would get painful to maintain.
We have discussed in the past about somehow "symbolizing" text ids, but we haven't reached a proper definitive answer for it.
There was one PR that tried to introduce centralized text defines, but it ended up dying: https://github.com/zeldaret/mm/pull/586 
It would be nice to revisit the concept of centralized named defines for texts.

Reply:
Are you asking me to remove this enum? I thought I'd be asked to make it.
I won't get impatient for it, but a centralized message repository should be made as soon as possible.
If you insist, I will remove it.

Reply:
Yes please remove the enum.